### ISSUE TITLE
Unable to render outside of a notebook context

### ISSUE THREAD

**Issue by cpsievert** [QuantStack/ipytree#56]
The [suggested way](https://ipywidgets.readthedocs.io/en/stable/embedding.html#python-interface) of getting standalone HTML from ipywidgets doesn't appear to work:

```python
from ipytree import Tree
from ipywidgets.embed import embed_minimal_html

w = Tree(stripes=True)
embed_minimal_html("index.html", w)
```

```shell
$ python3 -m http.server
```

<img width="918" alt="Screen Shot 2021-11-30 at 10 27 40 AM" src="https://user-images.githubusercontent.com/1365941/144087221-bbfa37b8-1c4f-4f44-8185-da859a070ccf.png">

**Comment by pnsvk** [comment]
Try the previous version.. it worked for me with voila.

`pip install ipytree==0.2.0`

**Comment by martinRenou** [comment]
This should be fixed for 0.2.2 (similar to #57)
